rdmo/projects/handlers/generic_handlers.py

```diff
@@ -0,0 +1,110 @@
+from django.contrib.auth.models import User
```

Comment:
I am not sold on this layer of generic handlers, but lets talk about this in the meeting. Maybe custom project signals would be better?